packages/airnode-node/src/config/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import { join } from 'path';
+import { readFileSync } from 'fs';
+import { mockReadFileSync } from '../../test/mock-utils';
+import dotenv from 'dotenv';
+import { references } from '@api3/airnode-protocol';
+import { loadTrustedConfig } from '../index';
+
+const AirnodeRrpV0Addresses: { [chainId: string]: string } = references.AirnodeRrpV0;
+const RequesterAuthorizerWithErc721Addresses: { [chainId: string]: string } = references.RequesterAuthorizerWithErc721;
+
+describe('config validation', () => {
+  const exampleConfigPath = join(__dirname, '../../config/config.example.json');
+  const exampleSecrets = dotenv.parse(readFileSync(join(__dirname, '../../config/secrets.example.env')));
+
+  it('loads the config and adds default contract addresses', () => {
+    const chainId = '4';
+    const invalidConfig = JSON.parse(readFileSync(exampleConfigPath, 'utf-8'));
+    invalidConfig.chains[0].id = chainId; // Need to use an actual chain not hardhat
+    delete invalidConfig.chains[0].contracts;
+    delete invalidConfig.chains[0].authorizers.crossChainRequesterAuthorizers[0].contracts;
+    delete invalidConfig.chains[0].authorizers.crossChainRequesterAuthorizersWithErc721[0].contracts;
+    delete invalidConfig.chains[0].authorizers.requesterAuthorizersWithErc721[0].RequesterAuthorizerWithErc721;
```

Comment:
Why did you populate those fields in the example config if you're then emptying them here?

Reply:
The authorizers that have been added to the config are objects with several fields- here I'm only deleting the optional contract field in each before then confirming via the test that the deployed address is added.